politeiawww/userwww.go

```diff
@@ -52,8 +53,16 @@ func (p *politeiawww) handleNewUser(w http.ResponseWriter, r *http.Request) {
 
 	reply, err := p.processNewUser(u)
 	if err != nil {
-		RespondWithError(w, r, 0, "handleNewUser: processNewUser %v", err)
-		return
+		switch err {
+		case user.ErrQuiesced:
```

Comment:
This should be handled in the `RespondWithError` function so we don't have to manually check it on every route.

Reply:
Actually, after thinking about this a little more, the correct way to handle this is going to be to manually check for `ErrQuiesced` after db calls that write data and converting it to a user error.  For example:

```
	err = p.db.UserNew(u)
	if err != nil {
		if err == user.ErrQuiesced {
			err = www.UserError{
				ErrorCode: www.ErrorStatusQuiesced,
			}
		}
		return nil, err
	}
```

You need to update all the db calls that write data to handle the quiesced error properly.